spec/recipes/install_spec.rb

```diff
@@ -25,9 +25,9 @@
   end
 
   it 'extracts downloaded Kafka archive' do
-    expect(chef_run).to run_execute('extract-kafka').with({
-      cwd: %(#{Dir.tmpdir}/kafka-build),
-    })
+    expect(chef_run).to run_execute('extract-kafka').with(
+      cwd: %(#{Dir.tmpdir}/kafka-build)
+    )
```

Comment:
Think this could be collapsed to a single line, i.e.
```ruby
expect(chef_run).to run_execute('extract-kafka').with(cwd: %(#{Dir.tmpdir}/kafka-build))
```